src/Lachain.Console/Application.cs

```diff
@@ -183,7 +183,9 @@ public void Start(RunOptions options)
                     .Where(key => !key.Equals(wallet.EcdsaKeyPair.PublicKey))
             );
             Logger.LogInformation("Block synchronization finished, starting consensus...");
-            consensusManager.Start(blockManager.GetHeight() + 1);
+            if (options.NoRestoreState)   Logger.LogInformation("NoRestoreState Option is set. Will not attempt to Restore State");
+            
+            consensusManager.Start(blockManager.GetHeight() + 1, !options.NoRestoreState);
```

Comment:
The option is there, so that we may force restart everything, if all else fails. ie. the current solution of simultaneous restarts can still be done with this option.